From 077c3d1679687470d9de542a1b295b2f71569728 Mon Sep 17 00:00:00 2001
From: Vincent Untz <vuntz@suse.com>
Date: Mon, 12 May 2014 15:06:08 -0400
Subject: [PATCH] apache2: Fix the status.conf template for SUSE

We need to explicitly allow 127.0.0.1, as localhost resolves to IPv6.
---
 chef/cookbooks/apache2/templates/suse/mods/status.conf.erb | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 chef/cookbooks/apache2/templates/suse/mods/status.conf.erb

diff --git a/chef/cookbooks/apache2/templates/suse/mods/status.conf.erb b/chef/cookbooks/apache2/templates/suse/mods/status.conf.erb
new file mode 100644
index 0000000..bdb85df
--- /dev/null
+++ b/chef/cookbooks/apache2/templates/suse/mods/status.conf.erb
@@ -0,0 +1,14 @@
+#
+# Allow server status reports generated by mod_status,
+# with the URL of http://servername/server-status
+#
+# see http://httpd.apache.org/docs-2.2/mod/mod_status.html
+#
+<IfModule mod_status.c>
+    <Location /server-status>
+        SetHandler server-status
+        Order deny,allow
+        Deny from all
+        Allow from localhost ipv6-localhost 127.0.0.1
+    </Location>
+</IfModule>
-- 
1.9.1

